Private/Add-TaskScheduledForwarder.ps1

```diff
@@ -9,27 +9,27 @@ function Add-TaskScheduledForwarder {
         $Argument = @('-windowstyle hidden', 'C:\Support\GitHub\PSWinReporting\Examples\Trigger.ps1', "-EventID $(eventID) -eventRecordID '$(eventRecordID)' -eventChannel '$(eventChannel)' -eventSeverity $(eventSeverity)")
 
     )
-    $XmlTemplate = "$((get-item $PSScriptRoot).Parent.FullName)\Templates\Template-ScheduledTask.xml"
-    #$xmlTemplate = "$($($(Get-Module -ListAvailable PSWinReporting)[0]).ModuleBase)\Templates\Template-ScheduledTask.xml"
-    if (Test-Path $xmlTemplate) {
-        Write-Color 'Found Template ', $xmlTemplate -Color White, Yellow
+    $XmlTemplate = Join-Path (Get-Item $PSScriptRoot).Parent.FullName 'Templates\Template-ScheduledTask.xml'
+    
+    if (Test-Path $XmlTemplate) {
+        $Logger.AddRecord("Found Template $XmlTemplate")
```

Comment:
I believe Add-TaskScheduleForwarder is executed by https://github.com/EvotecIT/PSWinReporting/blob/master/Examples/RunMe-AddSubscriptions.ps1 which requires a definition for Logger. Please expand examples with new Logger variable. Also (not sure if you already fixed this but if that $Logger variable is not there, prompt user to define it) so that when we release new version user will have to define it.